checkov/terraform/plan_runner.py

```diff
@@ -85,7 +85,7 @@ def get_entity_context_and_evaluations(self, entity):
 
     def check_tf_definition(self, report, runner_filter):
         for full_file_path, definition in self.tf_definitions.items():
-            scanned_file = f"/{os.path.relpath(full_file_path)}"
+            scanned_file = f"{os.path.sep}{os.path.relpath(full_file_path)}"
```

Comment:
@BrentSouza We're using this convention across all our runners, wouldn't change this for now

Reply:
Ah yeah this leaked into a commit from changes I was making to get the unit tests to pass on Windows. I'll revert it.

Sidenote: It seems there is a mix between code using `os.path` functions to get os specific paths and hardcoded `/`'s. That causes a bit of noise when running things from Windows. I did find a possible bug as a result of this behavior but I didn't want to overload this pull request.